lambda_function/aws/resources/elastic_cloud_compute_handler.py

```diff
@@ -0,0 +1,101 @@
+import json
+import logging
+from collections import OrderedDict
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+import boto3
+import consts
+import yaml
+from aws.resources.base_handler import BaseHandler
+from port.entities import create_entities_json, handle_entities
+
+logger = logging.getLogger(__name__)
+
+
+class EC2Handler(BaseHandler):
+    def handle(self):
+        for region in list(self.regions):
+            aws_ec2_client = boto3.resource('ec2', region_name=region)
+            logger.info(f"List EC2 Instance, region: {region}")
+            self.next_token = '' if self.next_token is None else self.next_token
+            while self.next_token is not None:
+                list_instances_params = self.selector_aws.get('list_parameters', {})
+                if self.next_token:
+                    list_instances_params['NextToken'] = self.next_token
+                try:
+                    response = aws_ec2_client.instances.all()
+                    instance_list = list()
+                    for instance in response:
+                        instance_list.append({"id": instance.id})
+
+                except Exception as e:
+                    logger.error(
+                        f"Failed list EC2 Instance, region: {region},"
+                        f" Parameters: {list_instances_params}; {e}")
+                    self.skip_delete = True
+                    self.next_token = None
+                    break
+
+                self._handle_list_response(instance_list, region)
+
+                #self.next_token = instance_list.get('NextToken')
```

Comment:
Commented for debug?

Reply:
Unlike the other API that returns paginated response, the ec2.instances.all() returns all the instances so we wouldn't need the pagination functionality